### PR TITLE
Feature/bank 11/be account endpoints

### PR DESCRIPTION
**Pull request by cescriba-2022278**
account endpoints were both created and modified for review

### CODE REVIEW

File: src/application/account/account.routes.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+          const LL = getTranslationFunctions(req.locale);
+          const account = await Account.findOne({ balance, tp_status: ACTIVE });
+          if (account) {
+            throw new AccountAlreadyExist(
+              LL.ACCOUNT.ERROR.BALANCE_ALREADY_EXISTS(),
+            );
+          }
+
+          return true;
+        }),
```
Comment (bcastillo-2022474) [inline src/application/account/account.routes.js]: that middleware should not exist

```suggestion
```

File: src/application/account/account.routes.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+            tp_status: ACTIVE,
+          });
+          if (account) {
+            throw new AccountAlreadyExist(
+              LL.ACCOUNT.ERROR.BALANCE_ALREADY_EXISTS(),
+            );
+          }
+
+          return true;
+        }),
```
Comment (bcastillo-2022474) [inline src/application/account/account.routes.js]:
```suggestion
```

File: src/application/account/account.routes.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            tp_status: ACTIVE,
+          });
+          if (account) {
+            throw new AccountAlreadyExist(
+              LL.ACCOUNT.ERROR.OWNER_ALREADY_EXISTS(),
+            );
+          }
+
+          return true;
+        }),
```
Comment (bcastillo-2022474) [inline src/application/account/account.routes.js]:
```suggestion
```

File: tsconfig.json [hunk 4]
```diff
@@ -6,6 +6,6 @@
         "outDir": "./dist",
         "moduleResolution": "nodenext",
     },
-    "include": ["./src/**/*", "./test/**/*"],
+    "include": ["./src/**/*", "./test/**/*", "tests/account"],
```
Comment (bcastillo-2022474) [inline tsconfig.json]: should not touch tsconfig.json

```suggestion
    "include": ["./src/**/*", "./test/**/*"],
```

**Comment by cescriba-2022278** [comment]
modified account endpoints

**Review by bcastillo-2022474 — APPROVED** [review]
made lots of modifications, but I already fix them, just take a look to them

**Pull request merged**